prow/plugins/config.go

```diff
@@ -115,6 +115,11 @@ type Blunderbuss struct {
 	// insufficient reviewers are available. If ExcludeApprovers is true,
 	// approvers will never be considered as reviewers.
 	ExcludeApprovers bool `json:"exclude_approvers,omitempty"`
+	// UseStatusAvailability controls whether blunderbuss will consider GitHub's
+	// status availability when requesting reviews for users. This will use at one
+	// additional token per successful reviewer (and potentially more depending on
+	// how many busy reviewers it had to pass over).
+	UseStatusAvailability bool `json:"use_status_availability,omitempty"`
```

Comment:
Seems like we eventually want this to be default. So I'd be inclined to a) make this `IgnoreStatusAvailability` and b) add a EnableStatusAvailability flag that defaults to false, later defaulting to true, then removing the flag.

(Alternatively make IgnoreStatusAvailability a *bool that we make == true when nil for a while

Reply:
I'm not 100% sure we want the prow default to be on. It definitely will consume more tokens to be on, but not sure how many yet. I'd like to have this on for Kubernetes tho. Open to thoughts for/against this being default on for prow.

Reply:
I'm in favor of defaulting to on, initially gating with a feature flag, but we can migrate later.

This couples the default behavior and whether to enable the feature.